### PR TITLE
Added keywords beginning w/ L.

### PR DESCRIPTION
**Pull request by stefoss23**
Added keywords beginning w/ L.

### CODE REVIEW

**Comment by stefoss23** [comment]
jenkins build this please

File: opm/simulators/flow/MissingFeatures.cpp [hunk 1]
```diff
@@ -365,14 +365,44 @@ namespace MissingFeatures {
...
+            "LWSOWCR",
+            "LWSWCR",
+            "LWSWL",
+            "LWSWLPC",
+            "LWSWU",
+            "LX",
+            "LY",
+            "LZ",
             "MAPUNITS",
+            "MATCORR",
```
Comment (akva2) [inline opm/simulators/flow/MissingFeatures.cpp]: sneaky or accident?

**Comment by stefoss23** [comment]
jenkins build this please

**Comment by blattms** [comment]
backported to release via #2080

**Pull request merged**